driver.py

```diff
@@ -0,0 +1,201 @@
+from abc import ABC, abstractmethod
+from processors.drop_close import DropClose
+from processors.outlier_removal import GaussOutlierRemoval
+from processors.distance import Distance
+from processors.step_range import StepRange
+from processors.processor import Processor
+from data_loaders.ecomapper_loader import EcomapperLoader
+from data_loaders.data_loader import DataLoader  
+from data_loaders.json_loader import JsonLoader
+from models.gpr import GPR
+from models.svr import SV
+from models.model_manager import ModelManager
+from models.model import Model
+from visualizers.visualization import Visualization
+from visualizers.two_d_vis import TwoDVis
+from visualizers.three_d_vis import ThreeDVis
+import matplotlib as mpl 
+from matplotlib import pyplot as plt
+import argparse
+
+
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--json', '-j', help = "name of json file")
+parser.add_argument('--file', '-f', required = True, help = "name of csv file")
+parser.add_argument('--extra_data', '-e', required = True, help = "name of column of user-selected data")
+parser.add_argument('--delimiter', '-d', help = "delimiter separating the column names in the file (make this first in json file)")
+parser.add_argument('--latitude', '-l', help = "name of latitude column")
+parser.add_argument('--longitude', '-lo', help = "name of longitude column")
+parser.add_argument('--depth', '-de', help = "name of depth column")
+parser.add_argument('--current_step', '-c', help = "name of current step column")
+parser.add_argument('--dist_tol', '-dt', help = "tolerance for points being too close together on chart")
+parser.add_argument('--outlier_tol', '-o', help = "tolerance for outliers (unit same as selected data type)")
+parser.add_argument('--first', '-fi', help = "first step in range being examined")
+parser.add_argument('--last', '-la', help = "last step in range being examined")
+parser.add_argument('--predictor', '-p', help = "whether or not to use a predictor model to interpolate (y/n)")
```

Comment:
Ok I can do that but let's say the user makes a json file and wants to define the predictor variable in the file; should they still write y/n there, or should I make it true or false, or should I make it so that if they put anything at all then it means true?

Reply:
It should be true false.

Reply:
I made it true false (strings)

Reply:
Json supports true/false literals, don't use strings.

Reply:
K I changed it to booleans